libs/language-server/src/stdlib/builtin-blocktypes/LocalFileExtractor.jv

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+/**
+* Extracts a `File` from the local storage.
```

Comment:
Confusing naming since `localstorage` exists in browsers. I'd speak of file system.

```suggestion
* Extracts a `File` from the local file system.
```